docs/source/architecture/sub_stakes.rst

```diff
@@ -0,0 +1,290 @@
+------------------------------------------
+
+Deposit and locking
+^^^^^^^^^^^^^^^^^^^
+*Methods* : ``deposit(uint256,uint16)``,  ``deposit(address,uint256,uint16)``,  ``lock(uint256,uint16)``
+
+To become a staker, NU tokens must be transferred to the ``StakingEscrow`` contract and locked using one of the ``deposit()`` methods. If the staker already has unlocked tokens within the account in the contract, then the stake can be locked using the ``lock()`` method. If successful, a new element will be created in the array of sub-stakes with the next period as the starting date, and the duration equal to the input parameter.
+
+
+**Example:**
+
+    A staker deposits 900 tokens:
+        * 1st sub-stake = 900 tokens starting from the next period and a locking duration of 5 periods
+
+    .. aafig::
+        :proportional:
+        :textual:
+
+            stake
+            ^
+            |
+         900|   +-------------------+
+            |   |        1st        |   period
+            +---+---+---+---+---+---+---->
+            + 0 + 1 + 2 + 3 + 4 + 5 + 6
+
+
+
+Lock prolongation
+^^^^^^^^^^^^^^^^^
+*Methods* : ``prolongStake(uint256,uint16)``
+
+In order to increase the staking reward, as well as the possibility of obtaining policies with a longer timeframe, stakers can increase the duration of their locked sub-stake using the ``prolongStake()`` method. The number of sub-stakes does not change, but the locked duration for the specified sub-stake will be increased.
+
+**Example:**
+
+    A staker prolongs sub-stake for 2 additional periods up to 7:
+		- Before: 
+			* 1st sub-stake = 900 tokens with locking duration of 5 periods
+		- After: 
+			* 1st sub-stake = 900 tokens with locking duration of 7 periods
+
+    .. aafig::
+        :proportional:
+        :textual:
+
+                         Before             
+
+            stake
+            ^
+            |
+         900+-----------------------+
+            |          1st          |   period
+            +---+---+---+---+---+---+---->
+            + 0 + 1 + 2 + 3 + 4 + 5 + 6      
+			
+			
+			
+                         After             
+
+            stake
+            ^
+            |
+         900+-------------------------------+
+            |              1st              |   period
+            +---+---+---+---+---+---+---+---+---->
+            + 0 + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8
+			
+			
+Sub-stake division
+^^^^^^^^^^^^^^^^^^
+*Methods* : ``divideStake(uint256,uint256,uint16)``
+
+If necessary, stakers can extend the locking duration for only a portion of their tokens in a sub-stake by using the ``divideStake()`` method. This method splits a sub-stake into two elements: the old sub-stake with the reduced locked amount and the new sub-stake with the specified amount. The new sub-stake has the specified locked amount and an extended lock duration, based on the specified number of periods, with the same start period as the old sub-stake.
+
```

Comment:
A fundamental question for me is: why would a staker want to divide her stake? What's the use case or goal behind that? Can we give an example or motivation?

Reply:
Also, substake division needs a link from the staking guide. See
https://github.com/nucypher/nucypher/commit/36204bd86cf692037dbc1bb0278be14700a50ae8

Reply:
This supposed to be an answer: `can extend the locking duration for only a portion of their tokens in a sub-stake`. Any suggestions how to improve?